sr_port/gdsfhead.h

```diff
@@ -4083,7 +4083,16 @@ MBSTART {														\
 			break;												\
 		}													\
 		else if ((CSA)->now_crit && !crit_stuck)								\
-		{													\
+		{	/* Assert that we do not hold the ftok or access semaphore as that can cause deadlocks		\
+			 * while we have crit and wait below for freeze to be lifted, since the process to lift		\
+			 * the freeze off (MUPIP FREEZE -OFF) would need access to the ftok semaphore.			\
+			 */												\
+			DEBUG_ONLY(unix_db_info	*udi;)									\
+															\
```

Comment:
Extra blank line? Also semi-colons outside macros if possible helps me a lot.

Reply:
The extra blank line is there to separate the local variable definition from the code (a convention we have). Fixed the semi-colon. Can never remember which one works for you :)